app/components/WrapHistory.vue

```diff
@@ -450,11 +450,11 @@ export default {
                 return result
             case 'btc':
             case 'tomobtc':
-                result = new BigNumber(amount).div(10 ** 8).toString(10)
+                result = new BigNumber(amount).div(10 ** 18).toString(10)
```

Comment:
Should not hard-code decimals, refer to https://github.com/tomochain/tomobridge/blob/d472106b6a2eaa875a7f96b69c0656116ba80d99/app/components/Wrap.vue#L522